pages/style-guide/prose.md

```diff
@@ -0,0 +1,111 @@
+<script>{
+    "title": "Prose Style Guide"
+}</script>
+
+## Formatting Conventions
+
+Articles about all content (sites, README, docs, etc) are authored with [GitHub Flavored Markdown](http://github.github.com/github-flavored-markdown/).
```

Comment:
Most sites use markdown, while the API site uses XML. Once again, if you don't know this kind of things, you probably aren't the right guy to write this up.